monorepo/monorepo-ws/apps/federation-gateway/src/configuration/utils.ts

```diff
@@ -0,0 +1,24 @@
+import { EnumOfErrorTypes, FederationGatewayError } from './errors/errors';
+import { Storage } from '@monorepo-ws/supergraph-manager';
+
+export const validStorage = (storageString: string) => {
+  try {
+    return Storage[storageString];
```

Comment:
určitě tohle selže? Storage jestli je pole a přijde sem neznámej string, tak výsledek bude undefined.